docker/docker.hcl

```diff
@@ -0,0 +1,17 @@
+group "default" {
+  targets = ["graphql-mesh"]
+}
+
+variable "PWD" {
+  default = "."
+}
+
+variable "context" {
+  default = "${PWD}/docker"
+}
+
+target "graphql-mesh" {
+  context = "${context}"
+  dockerfile = "${context}/Dockerfile"
+  platforms = ["linux/amd64", "linux/arm64"]
```

Comment:
no tags?

Reply:
hmm...I thought this could be configured here from the CI, but it doesn't seem like it, switching back
```yaml
          set: |
            ${{ steps.docker-bake-inputs.outputs.set }}
            graphql-mesh.tags+=ghcr.io/ardatan/graphql-mesh:${{ github.sha }}
            ${{ github.ref == 'refs/heads/main' && 'graphql-mesh.tags+=ghcr.io/ardatan/graphql-mesh:v1-preview' || '' }}

```

Reply:
hmm you can check this: https://github.com/kamilkisiela/graphql-hive/blob/main/docker/docker.hcl#L36-L49 
it needs to be part of the `hcl` file 

Env vars are injected from the CI into the HCL, and then in the HCL you define the tags.